h/accounts/views.py

```diff
@@ -587,6 +587,16 @@ def reset_password_link(request, reset_code):
     return request.route_url('reset_password_with_code', code=reset_code)
 
 
+@view_config(route_name='dismiss_sidebar_tutorial', request_method='POST',
+             renderer='json')
+def dismiss_sidebar_tutorial(request):
+    if request.authenticated_userid is None:
+        raise accounts.JSONError()
+    else:
```

Comment:
Minor point of style (which isn't going to stop me merging) but:

```
if condition:
    raise Something()
else:
    something_else()
```

is usually more readable without the `else:`, i.e.

```
if condition:
    raise Something()
something_else()
```